rossum/lib/api_client.py

```diff
@@ -355,6 +355,10 @@ def _get_annotation_polling(self, annotation: int) -> dict:
             click.echo(" Processing failed.")
         return annotation_object
 
+    def create_workspace(self, name: str, organization: str, metadata: Dict[str, Any]) -> dict:
+        data = {"name": name, "organization": organization, "metadata": metadata}
```

Comment:
The best way would be to do it the same way you did it for queues before:
`create_workspace(self, name: str, organization: str, metadata: Optional[Dict] = None) `

Reply:
it will be fixed in the next commit